sintr_worker/lib/basic_info.dart

```diff
@@ -97,3 +97,37 @@ String logKey, Stream<String> stream) async {
   result[basic_info_key] = [unknown_value];
   return result;
 }
+
+Future<Map<String, List<String>>> reduce(
+    String basicKey, Stream<String> stream) async {
+  if (!basicKey.startsWith(key_prefix)) throw 'unknown key: $basicKey';
+  var dateSuffix = basicKey.substring(key_prefix.length);
```

Comment:
I prefer to user split, rather than substring. It's slightly more brittle and therefore prone to report any coding errors.